src/util/globals.js

```diff
@@ -218,7 +218,11 @@ export const getMapTilesSettings = () => {
     return getExtension("mapTiles");
   }
   /* defaults */
-  const api = 'https://api.mapbox.com/styles/v1/trvrb/ciu03v244002o2in5hlm3q6w2/tiles/256/{z}/{x}/{y}?access_token=pk.eyJ1IjoidHJ2cmIiLCJhIjoiY2l1MDRoMzg5MDEwbjJvcXBpNnUxMXdwbCJ9.PMqX7vgORuXLXxtI3wISjw';
+  let access_token = "no_access_token";
+  if (process.env.MAPBOX_API_TOKEN !== undefined) access_token = process.env.MAPBOX_API_TOKEN;
+  console.log(process.env.MAPBOX_API_TOKEN);
```

Comment:
did you leave the console.logs in on purpose?

Reply:
fixed it. I printed to debug